ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPageObject.java

```diff
@@ -241,16 +243,18 @@ public void checkButtonStatus(String buttonTitle, String status) {
 
     public Condition conditionValueOf(String status) {
         Condition condition;
-        switch (status) {
-            case "Active":
-            case "Visible":
+        switch (status.toLowerCase()) {
+            case "active":
+            case "visible":
                 condition = visible;
                 break;
-            case "Disabled":
-            case "Inactive":
+            case "disabled":
+            case "inactive":
                 condition = disabled;
                 break;
             default:
+                log.error("Invalid status {} supplied, please check your feature files for typos", status);
```

Comment:
Please add the message in the `fail()` call as well, the logs are not visible in the cucumber html report.

Reply:
Thanks for mentioning that, I just noticed that it isn't included in the reports :D